docs/installation/linux/centos.md

```diff
@@ -139,54 +177,63 @@ makes the ownership of the Unix socket read/writable by the `docker` group.
 
 To create the `docker` group and add your user:
 
-1. Log into Centos as a user with `sudo` privileges.
+1. Log into your machine as a user with `sudo` or `root` privileges.
```

Comment:
Unfortunately, we have to deal with a broad range of user skills, and have seen that if you don't _literally_ tell people what to do, there's a number of issues opened on the issue tracker that things don't work 😞